reconstruction.py

```diff
@@ -331,15 +368,20 @@ def reconstruct(self,evrange=(-1,-1,-1)):
         setattr(options,'outFile','%s_run%05d_%s.root' % (options.outname, run, options.tip))
         
     if not hasattr(options,"pedrun"):
-        pf = open("pedestals/pedruns.txt","r")
+        pedname= 'pedruns.txt'
+        if options.tag == 'DataMango':
+             pedname = 'pedruns_MANGO.txt'
+        if options.tag == 'MC':
+             pedname = 'pedruns_MC.txt'
+        pf = open("pedestals/"+pedname,"r")
```

Comment:
in effetti potremmo chiamare in generale pedestals/pedruns_<detectorname>.txt prendendo il nome dal config file